src/components/Footer/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './styles.scss';
+
+export default function Footer() {
+  return (
+    <section className='footer-container'>
```

Comment:
Quick question, for html is `.tsx`, should we use double quote?
```suggestion
    <section className="footer-container">
```

Reply:
Yep, all JSX elements should be double quote instead.